opentelemetry-sdk/src/opentelemetry/sdk/metrics/processor.py

```diff
@@ -0,0 +1,30 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from abc import ABC, abstractmethod
+
+
+class Processor(ABC):
+    @abstractmethod
+    def process(self):
+        """
+        This is to be called when a metric FIXME complete this
+        """
+
+
+class MeasurementProcessor(Processor):
```

Comment:
I'm assuming the `MeasurementProcessor` will take an exporter as argument to the constructor?

Reply:
Could, be that is more an implementation detail at the moment.